src/main/java/org/junitpioneer/jupiter/DisableIfTestFails.java

```diff
@@ -20,18 +20,18 @@
 
 /**
  * Disables all remaining tests in a container if one of them failed.
- *
+ * <p>
```

Comment:
You always open a paragraph, but you don't close them. That's invalid HTML https://www.w3schools.com/html/html_paragraphs.asp

Note: Don't use <br>-Tag to create empty lines.